src/main/java/uk/gov/justice/hmpps/prison/security/AuthenticationFacade.java

```diff
@@ -45,10 +40,9 @@ public String getCurrentUsername() {
     }
 
 
-    public AuthSource getProxyUserAuthenticationSource() {
+    public AuthSource getAuthenticationSource() {
         final var auth = getAuthentication();
         return Optional.ofNullable(auth).
-                filter(a -> StringUtils.isNotBlank(MDC.get(PROXY_USER))).
```

Comment:
Needed to split out Proxy user from Auth source, which felt like odd bed-fellows anyway.